deepcell_tracking/isbi_utils.py

```diff
@@ -352,23 +381,25 @@ def benchmark_division_performance(trk_gt, trk_res, path_gt, path_res):
     trks = load_trks(trk_res)
     lineage_res, _, y_res = trks['lineages'][0], trks['X'], trks['y']
 
-    # Produce ISBI style text doc to work with
-    trk_to_isbi(lineage_gt, path_gt)
-    trk_to_isbi(lineage_res, path_res)
+    # Produce ISBI style array to work with
+    if path_gt is not None or path_res is not None:
+        warnings.warn("UserWarning: The 'path_gt` and 'path_res' arguments are deprecated.")
```

Comment:
[`warnings.warn` supports `DeprecationWarnings` as a category](https://docs.python.org/3/library/warnings.html#warnings.warn), so you can just warn the relevant text:

```suggestion
    if path_gt is not None or path_res is not None:
        warnings.warn('The `path_gt` and `path_res` arguments are deprecated.',
                      DeprecationWarning)
```